spec/features/dams_object_spec.rb

```diff
@@ -434,6 +435,10 @@
       visit catalog_index_path( { q: 'THsdtk', sort: 'title_ssi asc' } )
       expect(page).to have_selector('h3', :text => "Image File Test")
     end
+    it 'should index fulltext of complex object CSV file' do
+      visit catalog_index_path( { q: 'Testing CSV', sort: 'title_ssi asc' } )
+      expect(page).to have_content("Image File Test")
```

Comment:
:+1: that looks good. Will the following expect assertions need to change as a result?

- https://github.com/ucsdlib/damspas/blob/feature/fulltext_index/spec/features/dams_object_spec.rb#L432
- https://github.com/ucsdlib/damspas/blob/feature/fulltext_index/spec/features/dams_object_spec.rb#L436
- https://github.com/ucsdlib/damspas/blob/feature/fulltext_index/spec/features/dams_object_spec.rb#L440